data/lib/src/datasource/authentication_datasource.dart

```diff
@@ -79,4 +84,30 @@ class AuthenticationDataSource {
       throw UnknownError(exception.toString());
     }
   }
+
+  void _throwLogoutException(dynamic exception) {
```

Comment:
We have `RemoteExceptionThrower` class to throw common exception and so on. Please find usage of it to get more detail.

Reply:
Done! Thanks